app/models/chargeback_rate.rb

```diff
@@ -70,4 +72,9 @@ def self.seed
       end
     end
   end
+
```

Comment:
I agree with @imtayadeway here.  This is really a responsibility of the model. Is it true that any place we try to delete the model, we'll have to reproduce this logic?